region/norfair/crocomire/Post Crocomire Jump Room.json

```diff
@@ -384,9 +388,16 @@
         "h_canArtificialMorphBombHorizontally",
         "h_canArtificialMorphIBJ",
         "canSuitlessLavaDive",
-        {"acidFrames": 175}
+        {"acidFrames": 175},
```

Comment:
There is some finesse and variation to how to do this.  But basically trickyjump seemed more like the initial bomb jump with momentum, or perhaps single IBJ from acid.  And insanejump is where doubleIBJ from acid would fit.
My numbers are for whichever acid level I fell into, so some number of frames probably need to be added just in case.

- Bomb horizontally & roll right = 302
- Bomb with momentum or from an ibj & roll right = 267 (bounce on the kamer once then into the acid)
- Bomb from IBJ and then IBJ when close the solid platform = 227
- Bomb from IBJ and then double IBJ when close to the solid platform = 140
- Bomb from IBJ and then IBJ right away = 180 (stop on the kamer before falling in the acid for more control)
- Bomb from IBJ and then double IBJ right away = 119
- Bomb from IBJ and then IBJ while on the Kamer = 67

Reply:
Did a few tests to get a sense of the variance, based on the acid cycles.
Bomb horizontally and roll right [293, 267, 296] and starting from an IBJ [280, 272, 253]
Bomb horizontally and IBJ out [253, 192, 219] and starting from an IBJ [176, 197, 184]
Bomb horizontally and DIBJ out [237, 149, 168] and starting from an IBJ [152, 173, 171]

Of the 6 attempts, only the third of the one starting from an IBJ bounced on a Kamer. 
So, I'm thinking it should use the strats starting from an IBJ and assume missing the Kamer and moderately unlucky acid cycle; maybe 280 for rolling out, 220 for IBJing in the acid, and 175 for DIBJing in the acid?